Holovibes/sources/compute/image_accumulation.cc

```diff
@@ -67,7 +67,7 @@ void ImageAccumulation::init()
     if (setting<settings::XY>().output_image_accumulation > 1)
     {
         auto new_fd = fd_;
-        new_fd.depth = setting<settings::ImageType>() == ImgType::Composite ? 3 * sizeof(float) : sizeof(float);
+        new_fd.depth = static_cast<camera::PixelDepth>(setting<settings::ImageType>() == ImgType::Composite ? 3 * sizeof(float) : sizeof(float));
```

Comment:
You can change the two `sizeof(float)` in this line

Reply:
Fixed in last commit